### ISSUE TITLE
Streaming predict

### ISSUE THREAD

**Issue by khanifah-gif** [DigitalPhonetics/IMS-Toucan#203]
Hello @Flux9665 . I am currently exploring Toucan TTS. Thanks for ur great work at Toucan TTS. I wanna ask u, does Toucan TTS support streaming prediction? If it does, how can we enable it and if not, what steps are needed to make it support streaming prediction?

**Comment by Flux9665** [comment]
No, ToucanTTS is a fully non-autoregressive system. The entire audio is produced in parallel, so streaming is not possible. The entire input-sequence needs to be available in order to produce consistent prosody.

**Comment by khanifah-gif** [comment]
> No, ToucanTTS is a fully non-autoregressive system. The entire audio is produced in parallel, so streaming is not possible. The entire input-sequence needs to be available in order to produce consistent prosody.

Ahhh I see. Now I'm still exploring toucan. It is possible to increase the batch size for inference, so the processing time can be faster? @Flux9665

**Comment by johnwick123f** [comment]
@khanifah-gif I found lowering n_timesteps to little as 1 can increase speed by a large amount. Not sure if their is a quality decrease but overall seemed pretty similar quality.

https://github.com/DigitalPhonetics/IMS-Toucan/blob/MassiveScaleToucan/Modules/ToucanTTS/InferenceToucanTTS.py#L223
